docs/data/stashcache/run-stashcache-container.md

```diff
@@ -14,52 +15,55 @@ Before Starting
+1. **Docker:** For the purpose of this guide, the host must have a running docker service
+   and you must have the ability to start containers (i.e., belong to the `docker` Unix group).
 1. **Network ports:** Stash Cache listens for incoming HTTP/S connections on port 8000 (by default)
 1. **File Systems:** Stash Cache needs a partition on the host to store cached data
 
-Configuring Stashcache
-----------------------
+Configuring Stash Cache
+-----------------------
 
-In addition to the required configuration above (ports and file systems), you may also configure the behavior of your cache with the following variables using an enviroment variable file:
+In addition to the required configuration above (ports and file systems),
+you may also configure the behavior of your cache with the following variables using an environment variable file:
 
-Where the environment file on the docker host, `/opt/xcache/.env`, has (at least)the following contents:
+Where the environment file on the docker host, `/opt/xcache/.env`, has (at least) the following contents
+(replace `YOUR_SITE_NAME` with the name of your site as
+[registered in Topology](/data/stashcache/install-cache#registering-the-cache)):
 ```file
-XC_RESOURCENAME=ProductionCache
+XC_RESOURCENAME=YOUR_SITE_NAME
 XC_ROOTDIR=/cache
 ```
 
-### Optional Configuration ###
+### Optional configuration ###
 
-Further behaviour of the stashcache can be cofigured by setting the followin in the enviroment variable file
+Further behavior of the cache can be configured by setting the following in the environment variable file:
 
-- `XC_SPACE_HIGH_WM`: High watermark for disk usage;
-      when usage goes above the high watermark, the cache deletes until it hits the low watermark
-- `XC_SPACE_LOW_WM`: Low watermark for disk usage;
-      when usage goes above the high watermark, the cache deletes until it hits the low watermark
+- `XC_SPACE_HIGH_WM`, `XC_SPACE_LOW_WM`: High-water and low-water marks for disk usage;
+      when usage goes above the high-water mark, the cache will delete files until it hits the low-water mark.
 - `XC_PORT`: TCP port that XCache listens on
 - `XC_RAMSIZE`: Amount of memory to use for storing blocks before writting them to disk. (Use higher for slower disks).
-- `XC_BLOCKSIZE`: The size of the blocks in the cache
+- `XC_BLOCKSIZE`: Size of the blocks in the cache.
 - `XC_PREFETCH`: Number of blocks to prefetch from a file at once.
-       This is a value of how aggressive the cache is to request portions of a file. Set to `0` to disable
+       This controls how aggressive the cache is to request portions of a file. Set to `0` to disable.
 
-### Disabling OSG monitoring ###
+### Disabling OSG monitoring (testing only) ###
+
+!!! warning
+    Only disable OSG monitoring on services that are solely used for testing.
```

Comment:
I think @brianhlin suggested to get rid of this for the origin so same thing for the cache.

Reply:
Done